src/Entity/User/User.php

```diff
@@ -30,6 +38,21 @@ class User extends BaseUser
      */
     private $facebookId;
 
+    /**
+     * @ORM\Column(type="string", length=180, nullable=false)
+     */
+    private $email;
+
+    /**
+     * @ORM\Column(type="string", length=180, nullable=false)
+     */
+    private $nickname;
+
+    /**
+     * @ORM\Column(type="integer", length=1)
```

Comment:
Why not bool?
boolean: Type that maps a SQL boolean or equivalent (TINYINT) to a PHP boolean.

Reply:
It was copy/paste from the bundle. Fixed in https://github.com/trackhub/website/pull/202/commits/8f90234fd3620104d5f7349bcd8288e27d365260